### PR TITLE
Added video-column block and tweaked narrower style

### LINKED ISSUE #89: Business Performance page
https://www.sunstar.com/about/business-performance

### PR DESCRIPTION
**Pull request by dnbute**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #89 

- Added `video-column` block
- Tweaked `narrower` style
- Added `image-with-title` variant for `image-collage` block
- Tweaked image caption autoblocking

Test URLs:
- Original: https://www.sunstar.com/about/business-performance/
- Before: https://main--sunstar--hlxsites.hlx.live/about/business-performance
- After: https://business-performance--sunstar--hlxsites.hlx.live/about/business-performance

- [x]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

**Comment by dnbute** [comment]
> ### 🔸 1 visual difference detected
> * **/sidekick/blocks/sections** ([main](https://main--sunstar--hlxsites.hlx.page/sidekick/blocks/sections) vs [branch](https://business-performance--sunstar--hlxsites.hlx.page/sidekick/blocks/sections))Expected an image 1312px by 6560px, received 1312px by 6512px. 278083 pixels (ratio 0.04 of all image pixels) are different.
> 
> The diff images are [attached in the artifact](https://github.com/hlxsites/sunstar/actions/runs/6381832246)

Expected due to adding bottom margins to `narrower` style

**Review by bosschaert — APPROVED** [review]

**Comment by JiangLong2019** [comment]
Nit: Some small-size fonts need to be shown.
maybe this can be resolved by authoring, or it's ok if Dejan doesn't have any concerns about it ^_^
image

**Comment by dnbute** [comment]
> Nit: Some small-size fonts need to be shown. maybe this can be resolved by authoring, or it's ok if Dejan doesn't have any concerns about it ^_^ image

working on it!

**Comment by dnbute** [comment]
> > Nit: Some small-size fonts need to be shown. maybe this can be resolved by authoring, or it's ok if Dejan doesn't have any concerns about it ^_^ image
> 
> working on it 👍

Fixed; @JiangLong2019 mind taking a look at the page and approving this if it looks good? thanks

**Comment by JiangLong2019** [comment]
> > Nit: Some small-size fonts need to be shown. maybe this can be resolved by authoring, or it's ok if Dejan doesn't have any concerns about it ^_^ image

@dnbute Looks good to me, thanks!



BTW, when I opened the page I found there was a big white area between the content and footer, is this caused by authoring?

image

**Comment by dnbute** [comment]
> > > Nit: Some small-size fonts need to be shown. maybe this can be resolved by authoring, or it's ok if Dejan doesn't have any concerns about it ^_^ image
> 
> @dnbute Looks good to me, thanks!
> 
> BTW, when I opened the page I found there was a big white area between the content and footer, is this caused by authoring?
> 
> image
oh, that's new; I'll fix it and then merge, thanks!

File: styles/styles.css [hunk 1]
```diff
@@ -557,7 +558,7 @@ main .section.spaced h2 {
 }
 
 main .section.spaced h3 {
-  margin: 2rem 0;
+  margin: 2rem 0 0.5rem !important;
```
[the remark this answers is not in the document]
Comment (dnbute) [reply styles/styles.css:561]: I'll fix it in this PR
https://github.com/hlxsites/sunstar/pull/121

**Pull request merged**